account-management-api/src/main/java/uk/gov/di/accountmanagement/entity/NotificationType.java

```diff
@@ -45,14 +45,7 @@ public enum NotificationType implements TemplateAware {
     @Override
     public String getTemplateId(
             SupportedLanguage language, ConfigurationService configurationService) {
-        String templateId = configurationService.getNotifyTemplateId(getTemplateName(language));
-        if (!configurationService.isNotifyTemplatePerLanguage()
-                || templateId == null
```

Comment:
We're losing these two checks (for templateId == null and templateId.length() == 0), are we sure we don't need them?

Reply:
configurationService.isNotifyTemplatePerLanguage() was always false so it never hit this code anyway. Also this is part of the second commit which separates templateId and language so dependency is only on templateName now.